pages/about.html

```diff
@@ -0,0 +1,148 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <link rel="stylesheet" href="../css/mystyle.css">
+    <title>capStoneProject</title>
+  </head>
+  <body>
+    <div class="topHead">
+      <a href="https://www.facebook.com/faisal.mumtaz.3"><img src="../images/fb.png" alt="No Facebook icon found" width="10.12px" height="18px"/></a>
+      <a href="https://twitter.com/Faisalakb786"><img src="../images/twitter.png" alt="No twitter icon found" width="18px" height="18px"></a>
+      <h6>English</h6>
+      <h6>MyPage</h6>
+      <h6>Logout</h6>
+    </div>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light" >
+      <div class="container-fluid">
+
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarNav"
+          aria-controls="navbarNav"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon "></span>
+        </button>
+        <a class="navbar-brand logo  logoHead" href="../index.html">
+          <img src="../logo.png" alt="no logo found" width="80px" height="50px" >
+          <strong>Web World</strong>
+        
+      </a>
+        <div
+          class="collapse navbar-collapse"
+          id="navbarNav">
+          <ul class="navbar-nav items ">
+            <li class="nav-item liItems">
+              <a class="nav-link active navActive"  aria-current="page" href="../pages/about.html"
+                ><b>About</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Program</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Join</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Sponsor</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>News</b></a>
+            </li>
+          </ul>
+          <button class="btn btn-outline-success mainBtn" type="button">CC Campaign</button>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container-fluid abCont">
+      <div class="row fstHalf">
+        <img src="../images/bgpic.jpg" class="abtbgimg" alt="no img found" width="100%" >
+            
+            <img class="abimg" src="../images/abtxt.png" alt="no img found">
```

Comment:
- [OPTIONAL] I see that you used an image as the main title of the page. The recommended is to use the `h1` tag to wrap the main title for better SEO performance and HTML semantics. It would be nice if you create the title this way, by importing the same fonts used in the template instead of using an image.